framework/src/modules/chain/bft/bft.js

```diff
@@ -72,7 +73,7 @@ class BFT extends EventEmitter {
 
 		const loadFromHeight = Math.max(
 			finalizedHeight,
-			lastBlockHeight - this.constants.activeDelegates * 2,
+			lastBlockHeight - this.constants.activeDelegates * BFT_ROUND_THRESHOLD,
```

Comment:
Can we make it more readable, may be assign this value in initialiser. 

```js
this.constants = {
   activeDelegates, 
   startingHeight,
   twoRounds: activeDelegates * 2,
   threeRounds: activeDelegates * 3,
}
```

And then use it here...

```js
lastBlockHeight - this.constants.threeRounds,
```

This looks more clear while reading the code logic.

Reply:
Instead of having 2 rounds, 3 rounds, I would prefer to have the 2 different constants and used accordingly